crates/node-bindings/src/parcel/parcel.rs

```diff
@@ -25,37 +28,58 @@ impl ParcelNapi {
     let thread_id = std::thread::current().id();
     tracing::trace!(?thread_id, "parcel-napi initialize");
 
+    // Parcel Core Options
+    let mut parcel_options = ParcelOptions::default();
+
     // Wrap the JavaScript-supplied FileSystem
-    let fs = FileSystemNapi::from_options(&env, &options)?;
+    parcel_options.fs = FileSystemNapi::from_options(&env, &options)?;
+
+    // Assign Rust thread count from JavaScript
+    let js_threads = options.get_named_property::<JsUnknown>("threads")?;
```

Comment:
why don't we type `options` statically?

Reply:
Are we able to pass these properties in as options? I tried this initially but it failed to compile
```rust
#[napi]
pub struct ParcelOptions {
  fs: Option<FileSystemNapi>,
  rpc: JsFunction,
}
```

Reply:
Yes you should be able to pass `JsFunction` and `Option<JsObject>`

Reply:
then `options.fs.map(FileSystemNapi::new)`

Reply:
Updated